skimage/future/manual_segmentation.py

```diff
@@ -46,7 +46,7 @@ def manual_polygon_segmentation(image, alpha=0.4, return_all=False):
 
     Returns
     -------
-    labels : array of int, shape ([Q, ]M, N)
+    labels : array of int, shape ([P, ]M, N)
```

Comment:
If we are trying to unify all of the docstring shapes, then specific letters should have specific meanings. In this case, the leading dimension is not "planes", it is, in a sense, replicas of the same image.